### ISSUE TITLE
Reporting of machine name broken on Windows

### ISSUE THREAD

**Issue by richardlau** [nodejs/node-report#59]
Looks like the switch to use C++ streams in #48 has broken the reporting of the machine name on Windows: 

Before:
```
Machine: IBM826-R93F9G4
```

After:
```
Machine: 000000000218DFB8 000000000218DFD8
```

**Comment by richardlau** [comment]
The `sv101_name` and `sv101_comment` fields in the [SERVER_INFO_101 structure](https://msdn.microsoft.com/en-us/library/windows/desktop/aa370903%28v=vs.85%29.aspx) are pointers to wide strings and std::ostream just prints these as pointers.

**Comment by rnchamberlain** [comment]
Fix landed in master, PR #61, https://github.com/nodejs/node-report/commit/d9a544306182b3ab04e6878f55e9a0eca900de78

**Comment by rnchamberlain** [comment]
CI runs on Node 4, 6, 7, 8:
https://ci.nodejs.org/view/post-mortem/job/nodereport-continuous-integration/93/
https://ci.nodejs.org/view/post-mortem/job/nodereport-continuous-integration/94/
https://ci.nodejs.org/view/post-mortem/job/nodereport-continuous-integration/95/
https://ci.nodejs.org/view/post-mortem/job/nodereport-continuous-integration/92/

**Comment by rnchamberlain** [comment]
@richardlau OK to close this now? Fix is in node-report 2.1.1 - thanks
